ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -56,12 +60,19 @@ function ConfirmBannerAlert({ ownerId }: { ownerId: string }) {
 
 type IntlFunction = (str: string) => string;
 
-const getTitle = (t: IntlFunction, confirmation?: Confirmation) => {
+const getTitle = (
+  t: IntlFunction,
+  confirmation?: Confirmation,
+  isNFT?: boolean,
+) => {
   switch (confirmation?.type) {
     case TransactionType.contractInteraction:
       return t('confirmTitleTransaction');
     case TransactionType.tokenMethodApprove:
-      return t('confirmTitleApproveTransaction');
+      if (isNFT) {
```

Comment:
Minor, ternary?

Reply:
keeping it as is for now since it's the pattern in use in this file